Gruntfile.js

```diff
@@ -158,14 +191,19 @@ module.exports = (grunt) => {
     },
   };
 
+  const PACKAGE_JSON = grunt.file.readJSON('package.json');
   grunt.initConfig({
     pkg: PACKAGE_JSON,
     webpack: {
       buildProdAllFonts: prodAllFonts,
-      buildProdNoFonts: prodNoFonts,
       buildProdBravuraOnly: prodBravuraOnly,
       buildProdGonvilleOnly: prodGonvilleOnly,
       buildProdPetalumaOnly: prodPetalumaOnly,
+      buildProdNoFonts: prodNoFonts,
+      buildProdFontModuleBravura: prodFontModuleBravura,
+      buildProdFontModulePetaluma: prodFontModulePetaluma,
+      buildProdFontModuleGonville: prodFontModuleGonville,
```

Comment:
Another thing to add to our documentation on adding new fonts.

Reply:
Yeah, we definitely need to update the fonts docs because some of the older process is now outdated. With this branch, you can now build a font plugin separately, without recompiling VexFlow from source. I had a little free time yesterday and I built a font plugin for a tiny subset of the Leland SMuFL font. 

With the CJS builds, you just deploy your vexflow-font-leland.js alongside vexflow-core.js and it will load it on the fly when you call `await Vex.Flow.fetchMusicFont('Leland', <PATH_TO_FONT_PLUGIN_FILE>)`

Reply:
That's awesome!